webportal/src/app/job/job-view/job-table.component.ejs

```diff
@@ -7,7 +7,7 @@
       <th>ElapsedTime</th>
       <th>Retries</th>
       <th>Status</th>
-      <th>Delete</th>
+      <!-- <th>Delete</th> -->
```

Comment:
Later, we should change it to "Cancel" to stop job.

Reply:
Agree. I've put this into my own backlog.